From 5d4c24580fe79535395161ff05dbca2ed35bd7c6 Mon Sep 17 00:00:00 2001
From: torben-hansen <50673096+torben-hansen@users.noreply.github.com>
Date: Mon, 27 Mar 2023 21:36:53 -0700
Subject: [PATCH] Bump release version to v1.6.0 (#909)

Prepare new release by bumping release version number string to v1.6.0.
---
 crypto/fipsmodule/service_indicator/service_indicator_test.cc | 4 ++--
 include/openssl/base.h                                        | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/crypto/fipsmodule/service_indicator/service_indicator_test.cc b/crypto/fipsmodule/service_indicator/service_indicator_test.cc
index b6e187299..d9f243ee0 100644
--- a/crypto/fipsmodule/service_indicator/service_indicator_test.cc
+++ b/crypto/fipsmodule/service_indicator/service_indicator_test.cc
@@ -3899,7 +3899,7 @@ TEST(ServiceIndicatorTest, DRBG) {
 // Since this is running in FIPS mode it should end in FIPS
 // Update this when the AWS-LC version number is modified
 TEST(ServiceIndicatorTest, AWSLCVersionString) {
-  ASSERT_STREQ(awslc_version_string(), "AWS-LC FIPS 1.5.1");
+  ASSERT_STREQ(awslc_version_string(), "AWS-LC FIPS 1.6.0");
 }
 
 #else
@@ -3942,6 +3942,6 @@ TEST(ServiceIndicatorTest, BasicTest) {
 // Since this is not running in FIPS mode it shouldn't end in FIPS
 // Update this when the AWS-LC version number is modified
 TEST(ServiceIndicatorTest, AWSLCVersionString) {
-  ASSERT_STREQ(awslc_version_string(), "AWS-LC 1.5.1");
+  ASSERT_STREQ(awslc_version_string(), "AWS-LC 1.6.0");
 }
 #endif // AWSLC_FIPS
diff --git a/include/openssl/base.h b/include/openssl/base.h
index deced99f0..a3b72a534 100644
--- a/include/openssl/base.h
+++ b/include/openssl/base.h
@@ -216,7 +216,7 @@ extern "C" {
 // ServiceIndicatorTest.AWSLCVersionString
 // Note: there are two versions of this test. Only one test is compiled
 // depending on FIPS mode.
-#define AWSLC_VERSION_NUMBER_STRING "1.5.1"
+#define AWSLC_VERSION_NUMBER_STRING "1.6.0"
 
 #if defined(BORINGSSL_SHARED_LIBRARY)
 
